src/microbe_stage/PatchManager.cs

```diff
@@ -193,6 +213,8 @@ private void UpdateLight(Biome biome)
 
     private void UnmarkAllSpawners()
     {
+        GD.Print("Umarking all spawners.");
```

Comment:
Forgot to include in the review that this print doesn't add anything and should be removed.

Reply:
Fixed.